conda.recipe/run_test.sh

```diff
@@ -12,11 +14,11 @@ export TEST_PLATFORM=$(python -c "import sys; print('win' if sys.platform.starts
 export PYTHONHASHSEED=$(python -c "import random as r; print(r.randint(0,4294967296))") && echo "PYTHONHASHSEED=$PYTHONHASHSEED"
 env | sort
 conda info
-conda create -y -p ./built-conda-test-env python=3.9
+conda create -y -p ./built-conda-test-env python=3.${TEST_MINOR_VER}
 conda activate ./built-conda-test-env
 echo $CONDA_PREFIX
 [ "$CONDA_PREFIX" = "$PWD/built-conda-test-env" ] || exit 1
-[ $(python -c "import sys; print(sys.version_info[1])") = 9 ] || exit 1
-python -c '__requires__ = ["ruamel_yaml_conda >= 0.11.14"]; import pkg_resources' || exit 1
+[ $(python -c "import sys; print(sys.version_info[1])") = ${TEST_MINOR_VER} ] || exit 1
+#python -c '__requires__ = ["ruamel_yaml_conda >= 0.11.14"]; import pkg_resources' || exit 1
```

Comment:
Let's remove this line if it's not needed.

Reply:
Done